frontend/public/components/cluster-settings/cluster-operator.tsx

```diff
@@ -76,7 +77,7 @@ export const ClusterOperatorList: React.SFC = props => <List {...props} Header={
 const allStatuses = [
```

Comment:
Since it's related I wanted to ask why are we not working with [all the status types](https://github.com/openshift/api/blob/master/operator/v1/types.go#L131-L145). If we try to mask them and only show some, then `Updating` is a bit confusing since it can kinda refer to both `Progressing` and `Upgradeable`

Reply:
It's a good question. We've summarized status as one value, which isn't what's shown in the CLI. We might revisit this, but I think that should be a separate PR if we do.

For what it's worth, `PrereqsSatisfied` and `Upgradeable`  aren't shown by the CLI either.

```
❯ oc get clusteroperators
NAME                                 VERSION                        AVAILABLE   PROGRESSING   FAILING   SINCE
authentication                       4.0.0-0.ci-2019-04-17-095602   True        False         False     21h
cloud-credential                     4.0.0-0.ci-2019-04-17-095602   True        False         False     22h
cluster-autoscaler                   4.0.0-0.ci-2019-04-17-095602   True        False         False     22h
```

Note that all conditions are visible on the ClusterOperator details page in console with these changes, so you can see it there.

Reply:
> We might revisit this, but I think that should be a separate PR if we do.

👍